src/main/java/ogc/rs/database/DatabaseServiceImpl.java

```diff
@@ -939,7 +939,7 @@ public Future<JsonObject> postStacCollections(JsonArray collectionDataList) {
                         UUID id = UUID.fromString(collectionData.getString("id"));
                         String title = collectionData.getString("title");
                         String description =  collectionData.getString("description");
-                        String dateTimeKey = collectionData.getString("datetimeKey");
+                        String dateTime = collectionData.getString("datetime");
```

Comment:
Here also, you can pass the value `datetime` in query itself. The column name has to be `datetimeKey`, the value is `datetime`